content/docs/1.14/deployment.md

```diff
@@ -282,23 +286,24 @@ Shards and replicas are some configuration values to take special attention to,
 index creation. [This article](https://qbox.io/blog/optimizing-elasticsearch-how-many-shards-per-index) goes into
 more information about choosing how many shards should be chosen for optimization.
 
-#### Upgrade to Elasticsearch 7.x
+#### Upgrade Elasticsearch version
 
-The index mappings in Elasticsearch 7 are not backwards compatible with the older versions. 
-Therefore using Elasticsearch 7 with data created with older version would not work.
-Elasticsearch 6.8 supports 7.x, 6.x, 5.x compatible mappings. The upgrade has to be done
-first to ES 6.8, then apply data migration or wait until old daily indices are removed (this requires
-to start Jaeger with `--es.version=7` to force using ES 7.x mappings for newly created indices).
+Elasticsearch defines wire and index compatibility versions. The index compatibility defines
+the minimal version a node can read data from. For example Elasticsearch 7 can read indices
+created by Elasticsearch 6, however it cannot read indices created by Elasticsearch 5 even
+though they use the same index mappings. Therefore upgrade from Elasticsearch 6 to 7 does not require any
+data migration. However, upgrade from Elasticsearch 5 to 7 has to be done through Elasticsearch 6 and wait
+until indices created by ES 5.x are removed or explicitly reindexed.
 
-Jaeger by default uses Elasticsearch ping endpoint (`/`) to derive the version which is used
-for index mappings selection. The version can be overridden by flag `--es.version`.
+Refer to the Elasticsearch documentation for wire and index compatibility versions. Generally
+this information can be retrieved from root/ping REST endpoint.
 
-##### Data migration
+##### Reindex
 
-The data migration must be done on Elasticsearch 6.8.
+Manual reindexing can be used when upgrading from Elasticsearch 5 to 7 (through Elasticsearch 6)
+without waiting until indices created by Elasticsearch 5 are removed.
 
-1. Create index templates for Elasticsearch 7.x. Start Jaeger collector with flag `--es.version=7` or submit the template to API manually.
-2. Reindex all span and service indices to indices with new mapping. The new indices will have suffix `-1`:
+1. Reindex all span indices to new indices with suffix `-1`:
```

Comment:
Does this not require service indices to be reindexed now?

Reply:
There is a statement at the end mentioning other indices.